lib/index.d.ts

```diff
@@ -0,0 +1,11 @@
+export declare enum ProcessDataFlag {
```

Comment:
We will still want to keep `./typings/windows-process-tree.d.ts` and remove this file